deploy/dynamo/sdk/src/dynamo/sdk/cli/serving.py

```diff
@@ -141,8 +141,8 @@ def create_dependency_watcher(
         "$(CIRCUS.WID)",
     ]
 
-    if worker_envs:
-        args.extend(["--worker-env", json.dumps(worker_envs)])
+    if resource_envs:
+        args.extend(["--worker-env", json.dumps(resource_envs)])
```

Comment:
@mohammedabdulwahhab , @ishandhanani - still a little confused on the plural here, we are passing the full list of resource_envs to a variable that is worker-env -> singular - is that right?

its the full list for all workers that we pass here to a single process? - is this a global watcher watching all workers?